### PR TITLE
Change pipeline to make Makefile.coq

### PR DESCRIPTION
**Pull request by start974**
Change pipeline to make `Makefile.coq`, with make command in `build-documentation`.

### CODE REVIEW

File: .github/workflows/buid-documentation.yml [hunk 1]
```diff
@@ -21,14 +21,15 @@ jobs:
...
-        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+        #if: github.event_name == 'push' && github.ref == 'refs/heads/master'
         uses: coq-community/docker-coq-action@v1
         with:
           custom_image: ${{ matrix.image }}
           before_script: |
             startGroup "Create Makefile.coq and add permission"
-              make || true
-              sudo chown -R coq:coq .
+              sudo chown -R coq:coq Makefile
```
Comment (Zimmi48) [inline .github/workflows/buid-documentation.yml:30]: It doesn't work if you do `sudi chown -R coq:coq .` before `make Makefile.coq`?

File: .github/workflows/buid-documentation.yml [hunk 2]
```diff
@@ -21,14 +21,15 @@ jobs:
...
+        #if: github.event_name == 'push' && github.ref == 'refs/heads/master'
         uses: coq-community/docker-coq-action@v1
         with:
           custom_image: ${{ matrix.image }}
           before_script: |
             startGroup "Create Makefile.coq and add permission"
-              make || true
               sudo chown -R coq:coq .
+              make Makefile.coq
+              sudo chown -R coq:coq Makefile.coq
```
Comment (Zimmi48) [inline .github/workflows/buid-documentation.yml:32]: Cool, it works. Can you also try removing this line to look if it still does?
```suggestion
```

File: .github/workflows/buid-documentation.yml (hunk 1 above)
Comment (start974) [reply .github/workflows/buid-documentation.yml]: Thanks, that was the problem.

File: .github/workflows/buid-documentation.yml (hunk 2 above)
Comment (start974) [reply .github/workflows/buid-documentation.yml]: It's works, thanks.

**Pull request merged**